observatory_platform/telescopes/grid.py

```diff
@@ -169,6 +169,7 @@ class GridTelescope:
     """ A container for holding the constants and static functions for the GRID telescope. """
 
     DAG_ID = 'grid'
+    DATASET_ID = 'digital_science'
```

Comment:
We talked about grouping the data produced by provider, into BigQuery datasets here https://github.com/The-Academic-Observatory/observatory-platform/issues/196#issuecomment-691743804.

To be consistent with this, I thought that it makes sense to name the datasets after the organisation (or provider) that released the data. The digital_science dataset could also include [Figshare](https://figshare.com/).

Reply:
Yeah I think it's a good idea to group the data by provider. The only thing is that it would be nice to map the dataset/table names easily to the DAGs.

If you look at the list of DAGs in the Airflow UI and I see 'unpaywall', or 'grid' and then go to BigQuery, how would I know that I can find 'unpaywall' and 'grid' under 'our_research' and 'digital_science' respectively?

Maybe the names will get too long if we combine the dataset&table ids, but we could add the dataset id somewhere in the DAG description or task id?

Reply:
Yeah I can add some notes in the documentation.